includes/class-wcsatt-display.php

```diff
@@ -103,11 +112,10 @@ public static function convert_to_sub_product_options() {
 			$product_type = 'simple';
 		}
 
-		if ( $product_type != 'variable' ) {
-			$post_id = $product->id;
-		} else {
-			$post_id = $product->children['visible']['0']; // Gets the first enabled variation
-		}
+		// If the product is a variable product then dont display the subscription options
+		if ( $product_type == 'variable' ) return false;
```

Comment:
The reason for doing this is because the variation products need to display the subscription options via Javascript which needs to be appended to the single variation wrap without modifying the template files.